__tests__/app.test.js

```diff
@@ -60,13 +69,59 @@ describe("GET /api/articles/:article_id returns correct article", () => {
       msg: "An article with provided article ID does not exist",
     });
   });
-  test("returns 404 error if article number is non-number", async () => {
-    const results = await request(app).get("/api/articles/cheese").expect(404);
+  test("returns 400 error if article number is non-number", async () => {
+    const results = await request(app).get("/api/articles/cheese").expect(400);
+    expect(results.body.msg).toBe("Bad Request");
+  });
+  test("returns 404 error if endpoint incorrectly inputted", async () => {
```

Comment:
I'd maybe reword the description of this test. It's not necessarily that the endpoint was incorrectly inputted, it's more that the endpoint they were looking for was not found.